en/docs/install-and-setup/setup/si-setup/configuring-cluster-coordination.md

```diff
@@ -39,7 +39,7 @@ should be configured for **all the nodes** as follows:
 | `             enabled            `                               | Set this value to `             true            ` to enable cluster coordination for the node.                                                                                                                                                                                       | `             true/false            `                                                          |
 | `             groupId            `                               | The group ID is used to identify the cluster to which the node belongs. Nodes that belong to the same cluster must be configured with the same group ID.                                                                                                                             | `             group-1            `                                                             |
 | `             coordinationStrategyClass            `             | The clustering class to be used.                                                                                                                                                                                                                                                     | `             org.wso2.carbon.cluster.coordinator.rdbms.RDBMSCoordinationStrategy            ` |
-| `             strategyConfig > datasource            `           | The shared datasource to be used in the cluster. The datasource specified must be properly configured in the `             deployment.yaml            ` file. For detailed instructions to configure a datasource, see [Configuring Datasources](_Configuring_Datasources_) .        | `             WSO2_CARBON_DB            `                                                      |
+| `             strategyConfig > datasource            `           | The shared datasource to be used in the cluster. The datasource specified must be properly configured in the `             deployment.yaml            ` file. For detailed instructions to configure a datasource, see [Configuring Datasources]({{base_path}}/install-and-setup/setup/si-setup/configuring-data-sources)    .        | `             WSO2_CARBON_DB            `                                                      |
```

Comment:
Do we need the space added after the closing curly brace?

Reply:
Fixed [d7baf75](https://github.com/wso2/docs-apim/pull/7325/commits/d7baf7539c3a8b3574cfc6f35b50f9285f28b3a2)